### PR TITLE
Week 1 octavio sifuentes

### PR DESCRIPTION
**Pull request by priceymineral**
# Submitting Week 1 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [✅] Coding Challenges
- [✅] Coding Concepts
- [✅] Interview Questions

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:
Is there a comprehensive list of JS operators that return a boolean value? Is that what you were looking for in question 2 of the interview-questions? I couldn't really find one :(

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -57,6 +118,28 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+// create a fn that takes in an array and a number
+const lastIndex = (arr, num) => {
+    // loop from end to beginning of array
+    for (let i = arr.length - 1; i >= 0; i--) {
+        // if current number is equal to given number
+        if (arr[i] === num) {
```
Comment (TsunamiCode5) [inline code-challenges.js:126]: I really liked the way you did this! You were really thought outside of the box with this one!

File: interview-questions.md [hunk 2]
```diff
@@ -14,44 +14,44 @@ Researched answer:
...
-Your answer:
+Your answer: Git is a version control software to manage code modifications. Github is a web host for code that is managed via git commands.
 
-Researched answer:
+Researched answer: No significant differences between my and the researched answer.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: !!, Array.isArray, ==, ===, &&, ||, !=, !==,
```
Comment (TsunamiCode5) [inline interview-questions.md:23]: If you take a look at JavaScript Introduction section in the syllabus it covers this. Logical operators return boolean values. To include loose equality and strict equality (:

File: code-challenges.js [hunk 3]
```diff
@@ -31,6 +31,37 @@ const temp2 = 350
...
+
+console.log(isBoiling(temp1))
+console.log(isBoiling(temp2))
+console.log(isBoiling(temp3))
+
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
```
Comment (nmaresrivera11) [inline code-challenges.js:67]: Pseudo code is required

File: code-challenges.js [hunk 4]
```diff
@@ -39,13 +70,43 @@ const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
...
+    return combinedArray.length
+}
+
+console.log(combinedLength(padres1984WorldSeriesRuns, padres1998WorldSeriesRuns))
+
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
```
Comment (nmaresrivera11) [inline code-challenges.js:88]: ->

**Review by TsunamiCode5 — CHANGES_REQUESTED** [review]
Disregard (:

**Review by TsunamiCode5 — APPROVED** [review]
Octavio you did a really great job on this assessment! I loved your thinking out of the box method you used with the for loop in the code challenges. Your answers are really thought through. The interview questions are a good chance to practice answering real potential questions you may likely be asked one day in an interview. Good job explaining the researched answers into your own words, it will help solidify your understanding and quickness on the spot. This assessment comprehension level is advanced beginner, keep up the great work and forward thinking!

**Pull request merged**